Makefile

```diff
@@ -35,3 +41,89 @@ build-dependencies-axolotl:
 clean:
 	rm -f axolotl
 	rm -rf axolotl-web/dist
+
+check_platform_arm64:
+	@echo "Building Axolotl for Mobian (aarch64/amd64)"
+  ifneq ($(shell uname),Linux)
+	@echo "Platform unsupported - only available for Linux" && exit 1
+  endif
+  ifneq ($(shell uname -m),aarch64)
+	@echo "Machine unsupported - only available for aarch64/arm64" && exit 1
+  endif
+  ifneq ($(shell which apt),/usr/bin/apt)
+	@echo "OS unsupported - apt not found" && exit 1
+  endif
+
+dependencies_arm64:
+	@echo "Installing dependencies for building Axolotl..."
+	@sudo apt update
+	@sudo apt install golang nodejs npm python
+
+build_arm64:
+	@echo "Downloading (go)..."
+	@cd $(WORKDIR) && go mod download
+	@echo "Installing (npm)..."
+	@cd $(WORKDIR)/axolotl-web && npm install
```

Comment:
Using `npm ci` makes the build reproducible: the difference is that ci takes the exact versions from package-lock.json, and install an estimate. More here: https://docs.npmjs.com/cli/v6/commands/npm-ci#description

Reply:
`npm ci` seems to produce a more clean installation - that's indeed a better way. Thanks! I will test it and change the command.

Reply:
Apart from the reproducibility (if that's a word), a benefit of `ci` is that it never changes any files in the repo. npm install can sometime edit the package-lock file as to adjust the versions used within the range allowed (`^` or `~`).

https://semver.npmjs.com/

Reply:
Yes, I was just reading that part in the docs. Good idea!